src/mds/OpenFileTable.cc

```diff
@@ -58,47 +58,52 @@ void OpenFileTable::get_ref(CInode *in)
       ret.first->second.omap_idx = omap_idx;
     }
 
+    pin->recursive_opened_files++;
     in = pin;
   } while (in);
 }
 
 void OpenFileTable::put_ref(CInode *in)
 {
+  bool only_ajust_recursive_opened_files = false;
```

Comment:
never set only_ajust_recursive_opened_files to true.  what is it used for

Reply:
@ukernel Hi, sorry that I uploaded the wrong code, now it's right. Please take a look, thanks:-)